oak-search/src/test/java/org/apache/jackrabbit/oak/plugins/index/ExcerptTest2.java

```diff
@@ -0,0 +1,121 @@
+package org.apache.jackrabbit.oak.plugins.index;
+
+import com.google.common.base.Joiner;
+import org.apache.jackrabbit.api.JackrabbitSession;
+import org.apache.jackrabbit.commons.JcrUtils;
+import org.apache.jackrabbit.oak.api.PropertyValue;
+import org.apache.jackrabbit.oak.api.Result;
+import org.apache.jackrabbit.oak.api.ResultRow;
+import org.apache.jackrabbit.oak.plugins.index.search.FulltextIndexConstants;
+import org.apache.jackrabbit.oak.plugins.index.search.IndexFormatVersion;
+import org.apache.jackrabbit.oak.query.AbstractJcrTest;
+import org.junit.Test;
+
+import javax.jcr.Node;
+import javax.jcr.query.Query;
+import javax.jcr.query.QueryManager;
+import javax.jcr.query.QueryResult;
+import javax.jcr.query.RowIterator;
+
+import java.util.Iterator;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static com.google.common.collect.Sets.newHashSet;
+import static org.apache.jackrabbit.oak.api.QueryEngine.NO_BINDINGS;
+import static org.apache.jackrabbit.oak.api.Type.STRING;
+import static org.apache.jackrabbit.oak.plugins.index.IndexConstants.*;
+import static org.apache.jackrabbit.oak.plugins.index.IndexConstants.REINDEX_PROPERTY_NAME;
+import static org.junit.Assert.*;
+
+public abstract class ExcerptTest2 extends AbstractJcrTest {
+
+    protected TestRepository repositoryOptionsUtil;
+    protected Node indexNode;
+    protected IndexOptions indexOptions;
+
+    protected JackrabbitSession session = null;
+    protected Node root = null;
+
+    @Override
+    protected void initialize() {
+        session = (JackrabbitSession) adminSession;
+        try {
+            root = adminSession.getRootNode();
+            createContent();
+            adminSession.save();
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+
+    }
+
+    private void createContent() throws Exception {
+        Node def = JcrUtils.getOrAddNode(root, INDEX_DEFINITIONS_NAME).addNode("testExcerpt", INDEX_DEFINITIONS_NODE_TYPE);
+        def.setProperty(TYPE_PROPERTY_NAME, indexOptions.getIndexType());
+        def.setProperty(REINDEX_PROPERTY_NAME, true);
+        def.setProperty(FulltextIndexConstants.EVALUATE_PATH_RESTRICTION, true);
+        def.setProperty(FulltextIndexConstants.COMPAT_MODE, IndexFormatVersion.V2.getVersion());
+
+        Node properties = def.addNode(FulltextIndexConstants.INDEX_RULES)
```

Comment:
Right here, the properties node is created with `jcr:primaryType = oak:Unstructured` while in the original https://github.com/AngelaFabregues/jackrabbit-oak/blob/7234303f85a478b377ded3f64cf6c90c65beafdf/oak-search/src/test/java/org/apache/jackrabbit/oak/plugins/index/ExcerptTest.java#L79 the properties node has no `jcr:primaryType`.
This thing I'm currently assuming to have no impact at all.